.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,97 @@
+### CorsixTH Bug Report Template ###
+### Headers
+name: Bug report
+description: Create a report to help us improve
+title: "[Bug] Give a summary of the problem"
+labels: ["bug"]
+### Questions
+# What happened?
+body:
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Describe the issue
+      placeholder: Provide as much detail as you can
+    validations:
+      required: true
+# Reproducing Steps
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Steps to Reproduce
+      description: Break down as clearly as possible how to reproduce the error. If you cannot reproduce describe best how it happened
+      value: |
+        1. *Load save game attached to this ticket*
+        
+        2. 
+        
+        3.
+    validations:
+      required: true
+# Expected Behaviour
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected Behaviour
+      description: What did you expect to happen in the game?
+    validations:
+      required: true
+# Save Game Upload
+  - type: textarea
+    id: savegame
+    attributes: 
+      label: Save Game
+      description: |
+        It is often useful for us if you have a save game from shortly before the issue occurs that can be used to recreate the problem. Sometimes a save that shows the issue happening may be useful as well.
+        You can upload your save by putting it in zip file, then copy/pasting it directly into this text area. You can also click the "Attach files by dragging & dropping, selecting or pasting them" text at the bottom to upload the zip file.
+        Alternatively, you can use an external source such as Google Drive, Dropbox, or OneDrive and sharing the public link below to access your save game. The most important thing is to not remove these files after you upload them.
+      value: "Save game attached to report: Yes/No"
```

Comment:
I know we usually like to insert line breaks for readability of code. However, in YAML doing so would also insert those breaks into the output.

Reply:
I think you can use > (folding) instead of | (literal) in this instance line breaks should become spaces. New lines can be included by using two new lines instead

Reply:
Thanks I'll try that